Support overwriting existing save states

yoshisuga opened the pull request:
Show an alert controller when selecting a save state

yoshisuga commented on Provenance/Emulator/PVEmulatorViewController.swift:
```diff
@@ -721,45 +721,46 @@ class PVEmulatorViewController: PVEmulatorViewControllerRootClass, PVAudioDelega
 	func createNewSaveState(auto: Bool, screenshot: UIImage?) {
 		let saveFile = PVFile(withURL: URL(fileURLWithPath: saveStatePath).appendingPathComponent("\(game.md5Hash)|\(Date().timeIntervalSinceReferenceDate).svs"))
-		
-		var imageFile: PVImageFile?
```
yeah it looks like it - i didn't change any of the logic in create the image file and save state. i must have moved things around and put it back in place and did a ctrl-i on that entire block.

jasarien replied:
In future, we’d prefer not to have changes like that committed. It makes reviewing the PR more difficult.

yoshisuga replied:
understood - let me try to revert the changes for that section

jasarien replied:
That’s much better, thanks!

yoshisuga commented:
Ok, updated my branch and reverted that unchanged section in the `createNewSaveState` method

Looks like this happened because i was using soft-tabs (spaces instead of tabs) :(

jasarien commented:
Just gave this a test. Seems to work fine — couldn’t see any issues.
